tools/c7n_azure/c7n_azure/utils.py

```diff
@@ -301,3 +305,32 @@ def build_ports_dict(nsg, direction_key, ip_protocol):
                     ports[p] = IsAllowed
 
         return ports
+
+
+class AppInsightsHelper(object):
```

Comment:
Add tests for this class, especially around the different instrumentation key configs.